src/setting/settings.ts

```diff
@@ -417,7 +417,7 @@ export class VariousComplementsSettingTab extends PluginSettingTab {
 
     new Setting(containerEl)
       .setName("Min number of characters for trigger")
-      .setDesc("It uses a default value of Strategy if set 0.")
+      .setDesc("Setting the value to 0 does not mean the suggestion will be triggered without any inputted character. Depending on the Strategy you choose the actual value will be differ. Read the docs for more info.")
```

Comment:
@ooker777 
Thank you. It became very clear! What do you think of making a minor adjustment to the following?

---

Setting the value to 0 does not mean that suggestions will be triggered without any inputted characters. Depending on the "Strategy" you choose, the preset value will be used.

---

I will add a link to the documentation for "the preset value".

Reply:
Yes, "preset" is a better word. However can you modify the preset value? The word seems to imply that you can change it to your preference

Reply:
> However can you modify the preset value? The word seems to imply that you can change it to your preference

No, we can't change the value. 

How about the following?

```diff
- Depending on the "Strategy" you choose, the preset value will be used.
+ Depending on the "Strategy" you choose, a designated value will be used for each.
```

Reply:
Yes, I think that will work

Reply:
@ooker777 
OK. Please fix it and commit 🙏